ee/localserver/krypto-ec-middleware.go

```diff
@@ -91,18 +80,19 @@ func (e *kryptoEcMiddleware) Wrap(next http.Handler) http.Handler {
 			return
 		}
 
-		v := url.Values{}
-
 		newReq := &http.Request{
-			Method: "GET",
+			Method: http.MethodPost,
 			URL: &url.URL{
-				Scheme:   r.URL.Scheme,
-				Host:     r.Host,
-				Path:     cmdReq.Path,
-				RawQuery: v.Encode(),
+				Scheme: r.URL.Scheme,
+				Host:   r.Host,
+				Path:   cmdReq.Path,
 			},
 		}
 
+		if cmdReq.Body != nil && len(cmdReq.Body) > 0 {
+			newReq.Body = io.NopCloser(bytes.NewBuffer(cmdReq.Body))
+		}
+
```

Comment:
Hrm. This might be reasonable, but also seems like it merits a callout.

The post input to the middle way is this krypto format of `{box: <data>, future: stuff}`. But after we unpack it, we pass it with the body from the box. 

I think that's correct -- after all, what else would we do.

Reply:
yea, I don't know what else we would do =)

I'll leave some comment explaining, but I think the v2CmdRequestType is pretty obvious

```go
type v2CmdRequestType struct {
	Path string
	Body []byte
}
```

Reply:
I think it's correct. The _outer_ body is our krypto protocol silliness, the inner body is whatever is being smuggled